channeldb/forwarding_log.go

```diff
@@ -297,6 +297,10 @@ func (f *ForwardingLog) Query(q ForwardingEventQuery) (ForwardingLogTimeSlice, e
 		}
 
 		return nil
+	}, func() {
+		resp = ForwardingLogTimeSlice{
```

Comment:
since this is called first, can make the initial declaration `var resp ForwardingLogTimeSlice` for clarity

Reply:
done